presto-hive/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -113,7 +113,7 @@ public SemiTransactionalHiveMetastore(HdfsEnvironment hdfsEnvironment, ExtendedH
         this.hdfsEnvironment = requireNonNull(hdfsEnvironment, "hdfsEnvironment is null");
         this.delegate = requireNonNull(delegate, "delegate is null");
         this.renameExecutor = requireNonNull(renameExecutor, "renameExecutor is null");
-        this.skipDeletionForAlter = requireNonNull(skipDeletionForAlter, "skipDeletionForAlter is null");
```

Comment:
I personally prefer this minor refactor to be the first commit, before the actual changes.

Reply:
sure, happy to reorder :).